.github/workflows/custom-tabs-nightly.yml

```
@@ -77,4 +77,14 @@ jobs:
           asana-section: ${{ vars.GH_ANDROID_APP_INCOMING_SECTION_ID }}
           asana-task-name: GH Workflow Failure - Custom Tabs Flows (Robin)
           asana-task-description: The Custom Tabs nightly workflow has failed. See https://github.com/duckduckgo/Android/actions/runs/${{ github.run_id }}
-          action: 'create-asana-task'
\ No newline at end of file
+          action: 'create-asana-task'
+
+      - name: Adding as a release blocker
+        if: ${{ failure() }}
```

```suggestion
     if: ${{ failure() && steps.create-failure-task.outputs.taskId != '' }}
```

This would ensure that we only try to add the task if the previous step has succeeded. ie, there’s a task created or already existent. Otherwise if the task creation fails, we’ll try to add a task that doesn’t exist.